index.html

```diff
@@ -168,7 +168,28 @@
 
 										<div class="group">
 											<div class="ribbon-group">
-												<input type="checkbox" data-role="switch"  id="unitSwitch" >
+												<input type="checkbox" data-role="switch" name="unitSwitch" id="unitSwitch" onclick="changeUnits()" >
+												<script>
```

Comment:
@kyleines The implementation looks good. It would be great if we could move this script out of the html and into a JS file that is shared by both index.html and electron.html.

Reply:
Looks good. Thanks!

Reply:
@benb1n I moved the function to js/viewer.js.  Much cleaner!  Not sure why I didn't just do that 🤦‍♂️